magnolia-voting-app/src/main/resources/voting-contentApp/templates/pages/thanksTemplate.ftl

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+    [@cms.page /]
+    ${resfn.css("/webresources.*css")}
```

Comment:
Should we include all the css files?

Reply:
I will include the specific CSS file in the next commit